bower.json

```diff
@@ -42,6 +44,11 @@
     "spice-html5-bower": "himdel/spice-html5-bower#^0.1.5",
     "es6-shim": "^0.35.0",
     "phantomjs-polyfill": "^0.0.2",
-    "fetch": "^1.0.0"
+    "fetch": "^1.0.0",
+    "rxjs": "^4.1.0",
+    "rx-angular": "rx.angular#^1.1.3"
+  },
+  "resolutions": {
```

Comment:
There's problem with angular animate, bootstrap and others. If you remove angular from `vendor/assets/` you might find it as well. At least it happend for me after removing all assets from bowerr_components, I was asked what angular should be used so I picked 1.5.5 and added it to resolutions.